source/process/terminology.md

```diff
@@ -16,19 +36,21 @@ An obvious error in Mattermost software. Changes required to accomodate unsuppor
 
 #### Crimson Force Field
 
-When on screen text, particularly error messages, isn't helpful to its intended audience we call it a "Crimson Force Field" situation. This implies the text presented is not substantially better than using random words like "Crimson Force Field" to explain things, and may even be worse, if the text ends up misleading its audience. The words "Crimson Force Field" are intended to reproduce for the reader the feeling of confusion and awkwardness a user feels when software is cryptic and unclear. 
+When on-screen text, particularly error messages, may be confusing or unhelpful for its intended audience we call it "Crimson Force Field". 
+
+The purpose of on-screen text is to be helpful to the reader, its purpose is not merely "correctness". We use the term "Crimson Force Field" to reproduce the awkward, confused feeling a user experiences when software is cryptic and unclear, and yet "not incorrect". 
 
```

Comment:
Propose adding the story here if we are going to stick with "Crimson force field". I think the connection to a story is what helps us remember these terms. ie counter animation